front/cypress/integration/content_builder/mobile_preview_toggle.ts

```diff
@@ -0,0 +1,87 @@
+import { randomString } from '../../support/commands';
+
+describe('Content builder mobile preview', () => {
+  let projectId = '';
+
+  const getIframeBody = () => {
+    // get the document
+    return cy
+      .get('iframe')
+      .its('0.contentDocument')
+      .should('exist')
+      .its('body')
+      .should('not.be.undefined')
+      .then(cy.wrap);
+  };
+
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.getAuthUser().then((user) => {
+      const projectTitle = randomString();
+      const projectDescriptionPreview = randomString();
+      const projectDescription = 'Original project description.';
+      const userId = user.body.data.id;
+
+      cy.apiCreateProject({
+        type: 'continuous',
+        title: projectTitle,
+        descriptionPreview: projectDescriptionPreview,
+        description: projectDescription,
+        publicationStatus: 'published',
+        participationMethod: 'ideation',
+        assigneeId: userId,
+      }).then((project) => {
+        projectId = project.body.data.id;
+        cy.visit(`/admin/projects/${projectId}/description`);
+        cy.get('#e2e-toggle-enable-content-builder')
+          .find('input')
+          .click({ force: true });
+      });
+    });
+  });
+
+  beforeEach(() => {
+    cy.setAdminLoginCookie();
+  });
+
+  after(() => {
+    cy.apiRemoveProject(projectId);
+  });
+
+  it('shows saved description if there is no draft content', () => {
+    cy.visit(`/admin/content-builder/projects/${projectId}/description`);
+    cy.wait(10000);
+    cy.get('#e2e-draggable-single-column').dragAndDrop(
```

Comment:
Koen mentioned to me that the wait should never be used as it can cause flaky behavior (I didn't end up needing any in the previous test). Does it fail if this is taken out??